app/javascript/hyacinth_v1/components/projects/enabled_dynamic_fields/EnabledDynamicFieldForm.js

```diff
@@ -223,7 +228,9 @@ export const EnabledDynamicFieldForm = (props) => {
   const [disabled] = useState(formType !== 'edit');
```

Comment:
The use of `useState` doesn't seem necessary for this `disabled` variable, since it's just derived from the formType prop.  (I know that's not your change, but I just noticed it here and wanted to mention it.)

Maybe it would be better to just remove this line altogether and make use of formType since it's only used once in the entire functional component.